src/main/java/de/retest/recheck/review/ignore/io/Loaders.java

```diff
@@ -55,6 +59,8 @@
 			Pair.of( AllMatchFilter.class, new AllMatchFilterLoader() ), //
 			Pair.of( CacheFilter.class, new CacheFilter.FilterLoader() ), //
 			Pair.of( JSFilterImpl.class, new JSFilterLoader() ), //
+			Pair.of( InsertedFilter.class, new InsertedFilterLoader() ), //
+			Pair.of( DeletedFilter.class, new DeletedFilterLoader() ), //
```

Comment:
I think it is acceptable to write something like:
```
matcher: id=foo, deleted
```
but having some random line
```
attribute-regex=border-.*-color
deleted
pixel-diff=5px
inserted
```
seems quite odd.

I would either remove the global definition or add something along the line:
```
change=deleted
```
Maybe even to the element itself.

Reply:
I very much like `change=deleted` in any case. I'll change it to that!